Sources/HummingbirdTesting/RouterTestFramework.swift

```diff
@@ -108,7 +108,7 @@ struct RouterTestFramework<Responder: HTTPResponder>: ApplicationTestFramework w
                     head: .init(method: method, scheme: "http", authority: "localhost", path: uri, headerFields: headers),
                     body: stream
                 )
-                let logger = self.logger.with(metadataKey: "hb_id", value: .stringConvertible(RequestID()))
+                let logger = self.logger.with(metadataKey: "hb.id", value: .stringConvertible(RequestID()))
```

Comment:
```suggestion
                let logger = self.logger.with(metadataKey: "http.request.id", value: .stringConvertible(RequestID()))
```

Reply:
Used an `hb` prefix instead of http prefix as `http.request.id` is not in the Otel spec and we don't own the http namespace